session_0/word_processing.rb

```diff
@@ -2,13 +2,15 @@
 # any non-lower case alphabet (A..Z) to corresponding lower case
 # alphabet
 def lower_case(words)
-  raise NotImplementedError # TODO
+  output = []
+  words.each {|word| output.push(word.downcase)}
```

Comment:
See if map function can be used here: https://www.rubyguides.com/2018/10/ruby-map-method/